exporter/awsemfexporter/grouped_metric.go

```diff
@@ -52,7 +54,25 @@ func addToGroupedMetric(pmd *pdata.Metric, groupedMetrics map[interface{}]*Group
 			continue
 		}
 
+
 		labels := dp.Labels
+
+		if metricType, ok := labels["Type"]; ok {
+			if(metricType == "Pod" && config.CreateEKSFargateKubernetesObject){
+				err := addKubernetesWrapper(labels)
+				if err != nil{
+					logger.Warn("Issue forming Kubernetes Object", zap.Error(err))
+					return err
+				}
+			} else if (metricType == "Container" && config.CreateEKSFargateKubernetesObject){
+				err  := addKubernetesWrapper(labels)
+				if err != nil {
+					logger.Warn("Issue forming Kubernetes Object", zap.Error(err))
+					return err
+				}
+			}
```

Comment:
Am I missing something? We are doing same thing in the if and else if block.

Reply:
They are the same thing - I wrote it this way mostly so that it would be easier to make changes in the future - but I'm happy to change it in the future. Right now the difference between the two cases are implicitly handled in "addKubernetesWrapper" but that could change in the future.

Reply:
I would prefer to reduce the code here and make some additional comment in `addKubernetesWrapper` to explicitly call out the distict mechanims.